release/DEFAULT_RELEASE_BRANCH

```diff
@@ -1 +1 @@
-1-19
+1-20
```

Comment:
Should we make this change in a different PR?

Reply:
I'm fine with that